pkgs/development/python-modules/emoji/default.nix

```diff
@@ -16,7 +16,7 @@ buildPythonPackage rec {
     owner = "carpedm20";
     repo = pname;
     rev = "refs/tags/v${version}";
-    hash = "sha256-fnVY4KwiqvSVYijlDckLq6qDrBJj/rJGMwaQ1mMygek=";
+    hash = "sha256-unPd0Z9hKN1mNsHleytYaKsnnMhpUn5K0ByH8vHc0AU=";
   };
```

Comment:
The update was create with `nix-update`. @r-ryantm works without  `passthru.updateScript` but it can take quite some time till it pick up the latest details.

Reply:
Since `updateScript` is set in python-modules, there is no need to add it explicitly unless it is complicated.
https://github.com/NixOS/nixpkgs/blob/48c81e4615ea4c451c414028434660f014d50b56/pkgs/development/interpreters/python/mk-python-derivation.nix#L308-L310